tests/lib/hardware/node_base.py

```diff
@@ -77,6 +77,13 @@ def disk_detach(self, capacity):
         """
         pass
 
+    @abstractmethod
+    def get_volume_name(self, disk_name):
+        """
+        Get the volume name of an attached disk
+        """
+        pass
```

Comment:
We'll need to implement this in the AWS driver too (this is why Travis failed).

I'll push a commit to this branch to do so.

Reply:
Great.